plugins/net_plugin/net_plugin.cpp

```diff
@@ -2472,13 +2467,17 @@ namespace eosio {
                                       bool blk_applied, const fc::microseconds& blk_latency) {
       peer_dlog(c, "${d} block ${bn}:${id}.. latency ${l}ms",
                 ("d", blk_applied ? "applied" : "got")("bn", blk_num)("id", blk_id.str().substr(8,16))
-                ("l", blk_latency == fc::microseconds::maximum() ? 0 : blk_latency.count()/1000) );
+                ("l", blk_latency.count()/1000) );
       if( app().is_quiting() ) {
          c->close( false, true );
          return;
       }
       c->latest_blk_time = std::chrono::system_clock::now();
       c->block_status_monitor_.accepted();
+      if (blk_latency.count() < config::block_interval_us && c->peer_syncing_from_us) {
```

Comment:
Should we check `blk_latency == fc::microseconds::maximum()` here?

Reply:
`blk_latency` is always the actual block latency now.